src/errorcodes-net-cli/Program.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System.CommandLine;
 using errorcodes_net_cli.Extensions;
+using errorcodes_net_cli.Generators;
 using errorcodes_net_cli.Validations;
 
 var rootCommand = new RootCommand("ErrorCodes.Net command line tool");
 
 rootCommand.AddCommands([
-    new ValidateCommand()
+    new ValidateCommand(),
+    new GeneratorCommand()
 ]);
 
 await rootCommand.InvokeAsync(args);
 
 if (ErrorManager.Errors.Count == 0)
 {
-    Console.WriteLine("No errors detected!");
+    Console.WriteLine("Success");
```

Comment:
This was confusing me when it would output "No errors detected!" when validating that there were no duplicates. Make it more generic and representative of what it actually means